src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -46,6 +46,8 @@ internal partial class InteractiveWindow : IInteractiveWindow, IInteractiveWindo
         /// </remarks>
         private readonly UIThreadOnly _uiOnly;
 
+        internal InteractiveWindowClipboard InteractiveWindowClipboard { get; set; } = new SystemClipboard();
```

Comment:
@amcasey Will do with another PR. This is blocking my other work and the CI builds take forever to finish :)